### ISSUE TITLE
Bug - SimpleList: arialabel should be applied to inner ul element

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#8249]
**Describe the problem**
When passing in `aria-label` to the SimpleList component, the attribute gets applied to the outer div. This causes axe to flag it as aria labels aren't well supported on div elements.

The aria-label should instead be applied to the `ul` element that gets rendered instead (currently the isGrouped variation applies an aria-labelledby, so this fix would most likely only need to be applied at line ~77 of the SimpleList code:

https://github.com/patternfly/patternfly-react/blob/d68f302609a6abf8da34d1c33b153f604d6b329d/packages/react-core/src/components/SimpleList/SimpleList.tsx#L75-L78

**Comment by thatblindgeye** [comment]
@erkanercan you got it!

**Comment by thatblindgeye** [comment]
@erkanercan We're looking to get this in with our current milestone so just wanted to check in to see if you'd still be able to work on this issue?

**Comment by erkanercan** [comment]
Hi @thatblindgeye,
I'll provide a PR for this issue shortly, sorry for the latency.
